odl/solvers/functional/default_functionals.py

```diff
@@ -2262,8 +2277,10 @@ def __init__(self, functional, sigma=1.0):
         >>> l1_norm = odl.solvers.L1Norm(space)
         >>> smoothed_l1 = MoreauEnvelope(l1_norm)
         """
+        # HELP: Does this only makes sense, if 
+        #       functional.range == RealNumbers()? Otherwise raise Error?
```

Comment:
No, there's a definition adapted for complex numbers, where the minimum is taken over the real part of the inner product only. Leave this as it is for now.

Reply:
> Leave this as it is for now.

That is, do add the `range` parameter.